lib/data/external/AwsClient.js

```diff
@@ -42,7 +45,6 @@ class AwsClient {
                }
                // because of encryption the ETag here could be
```

Comment:
I don't think this comment is relevant anymore and we shouldn't save the etag.  Let's be consistent with what we do for the other backends.